docs/pages/apis/graphql.fr.mdx

```diff
@@ -0,0 +1,51 @@
+import { Callout } from "nextra-theme-docs";
+
+<Callout type="warning" emoji="⚠️">
+  L'API GraphQL est toujours en travaux et quelques fonctionnalités comme le filtre et le tri ne sont **pas encore** implémentées,  pour le moment ! Des changements sont en attente.
```

Comment:
```suggestion
  L'API GraphQL est toujours en travaux et quelques fonctionnalités comme le filtre et le tri ne sont **pas encore** implémentées, pour le moment! Des changements sont en attente.
```

Reply:
Like for the other suggestions about spaces ("!" also wait for spaces before and after), I'll wait for your confirmation to apply, or not ;)
There is too much spaces, so I'll have to commit a change, whatever you choose. I'll do this as soon as I see your answer